src/translate_c.zig

```diff
@@ -1538,6 +1539,22 @@ fn transImplicitCastExpr(
     }
 }
 
+fn isBuiltinDefined(name: []const u8) bool {
+    inline for (std.meta.declarations(c_builtins)) |decl| {
```

Comment:
Use `@hasDecl` to avoid the loop.

Reply:
Doesn't that require the name to be comptime known?

Reply:
ah right; I didn't comprehend that this needed to work at runtime.